src/index.js

```diff
@@ -0,0 +1,8 @@
+require(`../src/Airport.js`);
```

Comment:
strt using jasmine, start using classes for all objects to give the code more felxibility, try to stick to simple user stories, or break them up further, to make the coding as basic as possible to keep it flexible and less prone to mistakes/ errors